sematic/examples/add/BUILD

```diff
@@ -9,6 +9,10 @@ sematic_pipeline(
     dev = True,
     registry = "558717131297.dkr.ecr.us-west-2.amazonaws.com",
     repository = "sematic-dev",
+    bases = {
```

Comment:
Can you revert this change? We don't really want users using multiple base images